### PR TITLE
New Footer

### PR DESCRIPTION
**Pull request by TeemoCell**
New bit smaller footer, better formating and new usefull links.
Old Footer:
image

New Footer
image

Darkmode
image

### CODE REVIEW

**Comment by Aberdeener** [comment]
Looks nice, can the load timer remain black, though? And the Github icon should probably be black as well. Thanks!

**Comment by TeemoCell** [comment]
> Looks nice, can the load timer remain black, though? And the Github icon should probably be black as well. Thanks!

Done

**Review by Aberdeener — APPROVED** [review]

**Comment by Aberdeener** [comment]
Thanks, will wait for others to give input as well!

File: custom/panel_templates/Default/footer.tpl [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+            <div class="col-sm-5 mb-0">
+                <span>{'Y'|date} &copy; <a href="https://namelessmc.com/" target="_blank">NamelessMC</a></span>
+            </div>
+            <div class="col-sm-2 text-center mb-0">
+                <span class="font-size-sa">{$PAGE_LOAD_TIME}</span>
+            </div>
+            <div class="col-sm-5 mb-0">
+                <div class="float-right">
+                    <a class="ml-1" href="https://github.com/NamelessMC/Nameless" target="_blank" data-toggle="tooltip" data-placement="top" title="{$SOURCE}"><i class="fab fa-github fa-fw text-github"></i></a>
+                    <a class="ml-1" href="https://discord.com/invite/3wzUeGjX?" target="_blank" data-toggle="tooltip" data-placement="top" title="Discord"><i class="fab fa-discord fa-fw text-discord"></i></a>
```
Comment (Derkades) [inline custom/panel_templates/Default/footer.tpl:13]: or discird.gg/nameless :)

Comment (TeemoCell) [reply custom/panel_templates/Default/footer.tpl:13]: Oh, didn't know there is such a link for namelessMC.

**Comment by PadowYT2** [comment]
theres one small issue

https://user-images.githubusercontent.com/71085027/146004124-b9902257-5326-4343-bd06-3cdc5b93245b.mp4

**Comment by TeemoCell** [comment]
> 

Yeah i have that too sometimes but i can't do anything, atleast i didn't found a way to fix it.

**Comment by CqldFire** [comment]
looks good! to fix that issue should be able to use jquery and have an event fire when one of those buttons are clicked that immediately unfocuses the element

**Comment by CqldFire** [comment]
Few issues looking at it from the web designer perspective:
1. d-flex should be changed to row for better bootstrap compatibility
2. Mobile responsiveness on smaller screens like Chromebooks is not great, maybe patch that up, or implement a custom flexbox.
3. I would suggest more side padding on the footer
4. Add this code to fix the icon popover issue (template.php). I would add more specific classes, but this works
```
$(".footer .ml-1").click(function(){
    $(".footer .ml-1").blur();
});
```

**Comment by Aberdeener** [comment]
Thank you!

**Pull request merged**